blocks/market-insight/market-insight.js

```diff
@@ -84,35 +56,53 @@ function addSocialButtonEvent(button, block) {
   });
 }
 
-async function decorateCards(block, placeholders, previousNode) {
-  const results = await fetchMarketInsightMockData();
+async function decorateCards(block, placeholders, cardCount, previousNode) {
+  const queryObj = await fetchMarketInsightMockData();
+  const results = queryObj.map((el) => {
+    const elArr = el.PublishedOnDate.split(' ');
```

Comment:
For null safety, can `el.PublishedOnDate` be undefined?

Reply:
Good point, let me add the adjustment.